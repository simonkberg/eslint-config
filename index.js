module.exports = {
  parser: 'babel-eslint',
  extends: ['standard'],
  plugins: ['babel'],
  env: {
    browser: true,
  },
  rules: {
    'comma-dangle': [
      2,
      {
        arrays: 'always-multiline',
        objects: 'always-multiline',
        imports: 'always-multiline',
        exports: 'always-multiline',
        // Prevent trailing commas in functions as this is
        // still not widely supported.
        functions: 'never',
      },
    ],

    // Unfixable, incompatible with prettier.
    'no-mixed-operators': 0,

    // Makes prettier-eslint adapt the default behaviour for
    // curly spacing without enforcing it as an eslint rule.
    'object-curly-spacing': [0, 'always'],

    // Disable the default semi-rule and use the version from
    // the babel plugin, which works with class properties.
    semi: 0,
    'babel/semi': [2, 'never'],
  },
}
