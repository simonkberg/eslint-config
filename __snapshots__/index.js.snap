// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@simonkberg/eslint-config should match the snapshot 1`] = `
Object {
  "extends": Array [
    "/home/simon/www/simonkberg/eslint-config/node_modules/eslint-config-standard/index.js",
    "/home/simon/www/simonkberg/eslint-config/node_modules/eslint-config-prettier/index.js",
  ],
}
`;
